rules-topics/16_runecarving.md

```diff
@@ -0,0 +1,104 @@
+# Rune Carving
+
+## Rune Carving Products
+
+### Tier I
+
+#### Glowing Wand Stone 
+   >This creates a softly glowing stone that can be inset into the end of a carved wand. This stone should not emit more light than a typical physical representation for a Glowing Light spell. This item is consumed on use. Acts as per the Glowing Light Spell.
+#### Rune of Trap Compression
+>This rune is designed to help a would-be trap user to conceal their traps more effectively. To use this rune, the user must declare "Activate Trap Compression", and attach this tag to the Trap, along with the regular Trap tag. This effect allows the rep to be half the normal minimum size requirement, shrinking the minimum to 6"x3"x3". This item is consumed on use.
+#### Wand of Agility
+> These Wands are specially crafted items that empower the user to more efficiently use an ability, allowing them to repeat the same ability without expending additional Agility. At creation, a Wand of Agility is attuned to a specific ability of up to 8th level. The Tier of this recipe and materials required, are equal to the level of the ability. Immediately after using an Ability from their Agility Pool, the user may activate a Wand of the same ability by declaring "Activate <Ability Incant>" and using a second instance of this ability at no cost. You may not chain Wands off each other. This item is consumed on use.
+#### Wand of Magic
+>These Wands are specially crafted items that empower the user to more efficiently cast a spell, allowing them to repeat the same spell without expending additional mana. At creation, a Wand of Magic is attuned to a specific spell of up to 8th level. The Tier of this recipe and materials required, are equal to the level of the spell Immediately after casting a spell from their Mana Pool, the user may activate a Wand of the same spell by declaring "Activate <Spell Incant>" and casting a second instance of this spell at no cost. You may not chain Wands off each other. This item is consumed on use.
+#### Wand of Stamina
+> These Wands are specially crafted items that empower the user to more efficiently use an ability, allowing them to repeat the same ability without expending additional Stamina. At creation, a Wand of Stamina is attuned to a specific ability of up to 8th level. The Tier of this recipe and materials required, are equal to the level of the ability. Immediately after using an Ability from their Stamina Pool, the user may activate a Wand of the same ability by declaring "Activate <Ability Incant>" and using a second instance of this ability at no cost. You may not chain Wands off each other. This item is consumed on use.
```

Comment:
`8th level` -> `8th tier`
`level of the` -> `tier of the`

Missing a period after `spell` in third sentence of Wand of Magic.